src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -113,19 +127,36 @@ export class SkyRepeaterItemComponent implements OnDestroy {
     this.collapse.complete();
     this.expand.complete();
     this.inlineFormClose.complete();
+
+    this.ngUnsubscribe.next();
+    this.ngUnsubscribe.complete();
+
+    this.repeaterService.destroyItem(this);
   }
 
-  public headerClick() {
+  public headerClick(): void {
     if (this.isCollapsible) {
       this.updateForExpanded(!this.isExpanded, true);
     }
   }
 
-  public chevronDirectionChange(direction: string) {
+  public initializeItem(): void {
```

Comment:
Instead of making this a public method, callable by the wrapping repeater component, why not just call the service in the repeater item's `ngOnInit`?

Reply:
Done.